scripts/west_commands/blobs.py

```diff
@@ -121,16 +131,58 @@ def list(self, args):
     def ensure_folder(self, path):
         path.parent.mkdir(parents=True, exist_ok=True)
 
-    def fetch_blob(self, url, path):
+    def fetch_blob(self, args, blob):
+        url = blob['url']
+        path = Path(blob['path'])
+
+        cached_blob = None
+        sha256 = blob['sha256']
+
+        # collect all cache dirs from west config and env variable
+        cache_dirs = args.cache_dirs or config.get('blobs', 'cache-dirs', fallback='')
+        auto_cache_dir = args.auto_cache or config.get('blobs', 'auto-cache', fallback=None)
+
+        # convert all cache dirs to Path
```

Comment:
```suggestion
```
Comments should add information or structure, not paraphrase obvious one-liners.